pkgs/development/libraries/openexr/default.nix

```diff
@@ -1,4 +1,9 @@
-{ lib, stdenv, fetchurl, autoconf, automake, libtool, pkgconfig, zlib, ilmbase }:
+{ lib, stdenv, buildPackages, fetchurl, autoconf, automake, libtool, pkgconfig, zlib, ilmbase, }:
+
+let
+  # Doesn't really do anything when not crosscompiling
+  emulator = stdenv.hostPlatform.emulator buildPackages;
```

Comment:
I wonder how it happened that it is here almost an year and no one hit
```
/nix/store/xfghy8ixrhz3kyy6p724iv3cxji088dx-bash-4.4-p23/bin/bash ./b44ExpLogTable > b44ExpLogTable.h
./b44ExpLogTable: ./b44ExpLogTable: cannot execute binary file
```
becase `stdenv.hostPlatform.emulator` is not "Doesn't really do anything", it is `stdenv.runtimeShell` when there is no cross-compilation and "bash ./binaryfile" does not work